src/content/en/updates/2018/08/wasm-av1.md

```diff
@@ -0,0 +1,311 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: WebAssembly lets us extend the browser with new features. This article shows how to port the AV1 video decoder and play AV1 video in any modern browser.
+
+{# wf_published_on: 2018-08-21 #}
+{# wf_updated_on: 2018-08-21 #}
+{# wf_tags: webassembly,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: WebAssembly lets us extend the browser with new features. This article shows how to port the AV1 video decoder and play AV1 video in any modern browser. #}
+{# wf_blink_components: Blink #}
+
+# Extending the browser with WebAssembly {: .page-title }
+
+{% include "web/_shared/contributors/alexdanilo.html" %}
+
+One of the best things about [WebAssembly](https://webassembly.org/) is the ability experiment with new capabilities and implement new ideas before the browser ships those features natively (if at all).. You can think of using WebAssembly this way as a high-performance polyfill mechanism, where you write your feature in C/C++ or Rust rather than JavaScript.
```

Comment:
General nit: I'm surprised the linter hasn't complained about needing 80 character lines. I prefer this style, but you might get pushback.